package.json

```diff
@@ -53,7 +55,8 @@
   "scripts": {
     "test": "grunt test",
```

Comment:
@tumijacob Grunt used to run jshint on the code before running the tests. In order to keep the behaviour the same now that jshint is being removed you should add a pretest script which runs standard.